### DISCUSSION TITLE
Livewire 3 page component layout view not found

### DISCUSSION THREAD

**Discussion by raakkan** [livewire/livewire#6611] [1. Help]
im using livewire 3 i have this problem **Livewire page component layout view not found: [layouts.app]**

layout file in resources/views/components/layouts/app.blade.php

**Comment by danie-ramdhani** [comment]
change your livewire config like this:

```php
'layout' => 'components.layouts.app',
```

or move the app.blade to `resources/views/layouts/app.blade.php`

**Reply by raakkan** [reply]
``` 
'layout' => 'components.layouts.app',
```
this is livewire 3 default config and layout generated with php artisan livewire:layout this comment.

**Comment by raakkan** [comment]
this is my main problem button click work correctly but browser javascript console throws this error

```
livewire.js?id=11c49d7e:536     POST http://127.0.0.1:8000/livewire/update 500 (Internal Server Error)
```

this is livewire 3 problem and i also try this but no solution. 

```
Livewire::setUpdateRoute(function ($handle) {
    return Route::post('/custom/livewire/update', $handle);
});
```

**Reply by raakkan** [reply]
demo repo  [https://github.com/raakkan/livewire-3](https://github.com/raakkan/livewire-3) you can check your local machine

**Reply by apriyansahjero-2** [reply]
i have same problem

**Comment by ideadx** [comment]
Publish the livewire config and change as following: 

    'layout' => 'layouts.app',

**Comment by japjoh** [comment]
You might need to import the Livewite attribute for layout to use it

`use Livewire\Attributes\Layout`

**Comment by AlexLeoTz** [comment]
Run
```
php artisan livewire:layout 
```
then move app.blade.php from views/layouts to views/component/layouts

**Reply by yassin459dz** [reply]
you save my life

**Reply by DanielMontorfano** [reply]
Funciona, Gracias!
